week-6/Homework/mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,27 +1,48 @@
-function setAlarm() {}
+function setAlarm() {
+  let inputField = document.querySelector("#alarmSet");
+  let title = document.querySelector("#timeRemaining");
+  let counter = inputField.value;
+  let resetAlarm = document.querySelector("#reset");
+
+  let x = setInterval(function () {
+    let audioEl = document.querySelector("#myAudio");
+
+    let minutes = Math.floor(counter / 60)
+      .toString()
+      .padStart(2, "0");
+    let seconds = (counter - minutes * 60).toString().padStart(2, "0");
+
+    title.innerHTML = `Time Remaining: ${minutes}:${seconds}`;
+    resetAlarm.addEventListener("click", function () {
```

Comment:
By putting the code here it will run on every second. You could move it outside of this function to `Line 6` so this doesn't happen